src/general/HelpScreen.cs

```diff
@@ -94,20 +94,18 @@ private void BuildHelpTexts(string category)
     {
         var helpTexts = SimulationParameters.Instance.GetHelpTexts(category);
 
-        var middleIndex = Math.Ceiling(helpTexts.Messages.Count / 2.0f);
-
-        for (var i = 0; i < helpTexts.Messages.Count; i++)
+        foreach (var text in helpTexts.Messages)
         {
-            var message = TranslationServer.Translate(helpTexts.Messages[i]);
+            var message = TranslationServer.Translate(text.Message);
 
             var helpBox = HelpBoxScene.Instance();
-            helpBox.GetNode<Label>("MarginContainer/Label").Text = message;
+            helpBox.GetNode<CustomRichTextLabel>("MarginContainer/CustomRichTextLabel").ExtendedBbcode = message;
 
-            if (i < middleIndex)
+            if (text.Column == HelpText.TextColumn.Left)
             {
                 leftColumn.AddChild(helpBox);
             }
-            else
+            else if (text.Column == HelpText.TextColumn.Right)
             {
                 rightColumn.AddChild(helpBox);
             }
```

Comment:
I think there should be a GD.PrintErr as an else here warning that a help text doesn't have the column set.

Reply:
Added.